trikot-viewmodels/viewmodels/src/commonMain/kotlin/com/mirego/trikot/viewmodels/ViewModel.kt

```diff
@@ -23,4 +23,18 @@ interface ViewModel {
      * Action to execute when the view is tapped
      */
     val action: Publisher<ViewModelAction>
+    /**
+     * Return true if it should be exposed as an accessibility element.
+     * Note : `accessibilityLabel` or `accessibilityHint` must be specified (at least one) if this property is set to true or it will have no impact.
+     */
+    val isAccessibilityElement: Publisher<Boolean>
```

Comment:
I think `isAccessibilityElement` could be a get var where we combine both label and hint and return true if one emit.

```
val isAccessibilityElement:  get { accessibilityHint.combine(accessibilityLabel).map { true }.first().shared() .startWith(false) }
```

Reply:
As it is a rule (that at least one emit something) I would enforce this this way.

Reply:
Having used it in "real life", it happens quite a lot that we want to temporarily make an element inaccessible. An example is when we display a side panel and we want this panel to be the only accessible element, we can disable everything else. 
IMO it seems easier and straightforward to simply emit a new boolean instead of reinitializing the `accessibilityLabel` and the `accessibilityHint` every time we want to make something inaccessible.